packages/compass-indexes/src/components/search-indexes-table/search-indexes-table.tsx

```diff
@@ -174,11 +178,15 @@ export const SearchIndexesTable: React.FunctionComponent<
   onPollIndexes,
 }) => {
   useEffect(() => {
+    if (!isWritable) {
+      return noop;
```

Comment:
You don't need to return a function from effect if you don't have any clean up to do

Reply:
Fixing it!